io.github.Strozer2002.sparrow.yaml

```diff
@@ -20,10 +20,11 @@ modules:
       - mkdir /app/bin/
       - ln -s /app/sparrow/sparrow /app/bin/sparrow
       - install -Dm644 io.github.Strozer2002.sparrow.metainfo.xml /app/share/metainfo/io.github.Strozer2002.sparrow.metainfo.xml
+      - install -Dm644 icons/128x128/io.github.Strozer2002.sparrow.png /app/share/icons/hicolor/128x128/apps/io.github.Strozer2002.sparrow.png
       - install -Dm644 icons/256x256/io.github.Strozer2002.sparrow.png /app/share/icons/hicolor/256x256/apps/io.github.Strozer2002.sparrow.png
       - install -Dm644 io.github.Strozer2002.sparrow.desktop /app/share/applications/io.github.Strozer2002.sparrow.desktop
     sources:
       - type: archive
         url: "https://github.com/Strozer2002/sparrow-linux/releases/download/v1.0.0/sparrow-wallet-1.0.tar.xz"
-        sha256: 47b45817bdfe886e6ce0325bdec18f9baf43dae6dd0acc7cb61981f2dc370f88
+        sha256: 758399d037899a67804814cccb21d9b27afdc1e822a4c0e6de2de3705e43d9af
```

Comment:
retagging is BAD. New tarball = new version.

Reply:
okay, sorry